test/test_spectral_ops.py

```diff
@@ -623,6 +649,19 @@ def test_fftfreq_numpy(self, device, dtype):
                 actual = torch_fn(*args, device=device, dtype=dtype)
                 self.assertEqual(actual, expected, exact_dtype=False)
 
+    @skipCPUIfNoMkl
+    @skipCUDAIfRocm
+    @onlyOnCPUAndCUDA
+    @dtypes(torch.float, torch.double)
+    def test_fftfreq_out(self, device, dtype):
```

Comment:
Seems like we should make these OpInfos in the (near) future?

Reply:
Are there any `OpInfos` for tensor creation functions yet?

Reply:
Have to start somewhere.